imports/api/EloCalculator.js

```diff
@@ -23,21 +23,21 @@ export class EloCalculator {
 		else //if (player == this.game.north_player)
 			index = 3;
 
-		var expectedScores = this.expectedScores();
-		var adjustedScores = this.adjustedScores();
-
-		var playerElo = this.getPlayerElo(player);
-
-		if (playerElo < 1300)
-			k = 140;
-		else if (playerElo < 1500)
-			k = 120;
-		else if (playerElo < 1600)
-			k = 100;
-		else if (playerElo < 1700)
-			k = 80;
-		else
-			k = 60;
+		const expectedScores = this.expectedScores();
+		const adjustedScores = this.adjustedScores();
+
+		const playerElo = this.getPlayerElo(player);
+
+		switch(this.gameType) {
+		case Constants.GAME_TYPE.HONG_KONG:
+			gamesPlayed = Number(Players.findOne({ hongKongLeagueName: player }).hongKongGamesPlayed);
+			break;
+		case Constants.GAME_TYPE.JAPANESE:
+			gamesPlayed = Number(Players.findOne({ japaneseLeagueName: player }).japaneseGamesPlayed)
+			break;
+		}
+
+		const k = 100 - Math.min(gamesPlayed, 10) - Math.min(Math.max(gamesPlayed - 10, 0), 10) * 2;
```

Comment:
Minor change and I'll commit, can you comment what the expected function of this is?